packages/victory-bar/src/v37/victory-bar.stories.tsx

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable react/no-multi-comp */
 import * as React from "react";
 import VictoryBar from "./victory-bar";
-import VictoryChart from "victory-chart/lib/v37/victory-chart";
+import VictoryChart from "victory-chart/es/v37/victory-chart";
```

Comment:
Storybook uses webpack which uses our `es/**` files not `lib/**`. Our wireit config thus depends on `build:lib:esm` so this is the proper refererence.

Reply:
- [ ] Yo